repository_test.go

```diff
@@ -103,9 +103,10 @@ func createCommit(c *C, r *Repository) plumbing.Hash {
 	}
 
 	h, err := wt.Commit("test commit message", &CommitOptions{
-		All:       true,
-		Author:    &author,
-		Committer: &author,
+		All:               true,
+		Author:            &author,
+		Committer:         &author,
+		AllowEmptyCommits: true,
```

Comment:
I added AllowEmptyCommits flag because this `createCommit` function is used repeatedly in `TestMergeFF` and `TestMergeFF_Invalid`.

Let me know if we should take different approach.

Reply:
That's absolutely fine. :thumbsup: